service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/RepresentativesController.java

```diff
@@ -87,18 +84,8 @@ public AboutToStartOrSubmitCallbackResponse handleSubmitted(@RequestBody Callbac
 
     @PostMapping("/submitted")
     public void handleSubmittedEvent(@RequestBody CallbackRequest callbackRequest) {
-        CaseData caseDataBefore = mapper.convertValue(callbackRequest.getCaseDetailsBefore().getData(), CaseData.class);
-        CaseData caseData = mapper.convertValue(callbackRequest.getCaseDetails().getData(), CaseData.class);
-
-        List<Element<Representative>> representativesBefore = caseDataBefore.getRepresentatives();
-        List<Element<Representative>> currentRepresentatives = caseData.getRepresentatives();
-
-        List<Element<Representative>> representativeParties = representativeService
-            .getRepresentativePartiesToNotify(currentRepresentatives, representativesBefore);
-
         applicationEventPublisher.publishEvent(new PartyAddedToCaseEvent(
-            callbackRequest, requestData.authorisation(), requestData.userId(),
-            unwrapElements(representativeParties)));
+            callbackRequest, requestData.authorisation(), requestData.userId()));
```

Comment:
does it not brake fpl-661, that notification is sent to parties added to case, now it looks that  notification will be sent to users added earlier

Reply:
yes you are right, I will go back and fix this